services/ui-src/src/utils/autosave/autosave.ts

```diff
@@ -111,8 +111,14 @@ export const autosaveFieldData = async ({
       .filter((field: FieldInfo) => isFieldChanged(field))
       // determine appropriate field value to set and return as tuple
       .map(async (field: FieldInfo) => {
-        const { name, value, defaultValue, overrideCheck } = field;
-        const fieldValueIsValid = await form.trigger(name);
+        const { name, value, defaultValue, hydrationValue, overrideCheck } =
+          field;
+        let fieldValueIsValid = false;
+        if (value !== hydrationValue && hydrationValue !== undefined) {
+          fieldValueIsValid = await form.trigger(name);
+        } else {
+          fieldValueIsValid = true;
+        }
```

Comment:
Thanks @karla-vm for making me lose a couple hours to this one 😆. This handles a specific use case for triggering validation on autosave for Choicelist -> ChoiceList -> Any field here or even move nested fields. If you click the parent choice of a nested field and immediately click the next child parent choice of a nested field the child child (You follow me still?) choice would fire validation because it rendered AND its not in the DB yet. User's never been there before though so lets not trigger validation just yet, yeh?